src/cljs/project_scoper/routes.cljs

```diff
@@ -21,10 +21,11 @@
   ;; --------------------
   ;; define routes here
   (defroute "/" []
+    (set! (.-hash (.-location js/window)) "#/")
     (rf/dispatch [::events/set-active-view :start-view]))
-  (defroute "/type" []
+  (defroute "/:path/" []
```

Comment:
i wonder if we need the `/` at the end of `/:path/` i dont think its necessary but maybe im missing something...